core/config/general_config.go

```diff
@@ -410,6 +409,10 @@ EVM_ENABLED=false
 		}
 	}
 
+	if str := c.viper.GetString("MIN_OUTGOING_CONFIRMATIONS"); str != "" {
+		c.lggr.Errorf("WARNING: MIN_OUTGOING_CONFIRMATIONS has been removed and no longer has any effect. EVM_FINALITY_DEPTH is now used as the default for ethtx confirmations instead. You may override this on a per-task basis by setting `minConfirmations` e.g. `foo [type=ethtx minConfirmations=%s ...]`", str)
```

Comment:
"warning" is logged as error. Better to make it clear is it an error or a warning.